src/documentation/sdks/python3-support.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: section
+title: "Apache Beam: Python 3 Support"
+section_menu: section-menu/sdks.html
+permalink: /documentation/sdks/python3-support/
+---
+# Python 3 Support
+
+## Status
+
+Work is in progress to add Python 3 support to Beam.  Current goal is to make Beam codebase compatible both with Python 2.7 and Python 3.4.
```

Comment:
Specifically, I think this intro sentence is great for putting right in the section of the other page.